erna/stream_runner_local_output.py

```diff
@@ -44,11 +43,12 @@ def run(jar, xml, input_files_df, output_path, aux_source_path=None):
             return "fact-tools generated no output"
                     
         if output_path.endswith("gz"):
-            f_in = open(tmp_output_path, 'rb')
-            f_out = gzip.open(tmp_output_path+'.gz', 'wb')
-            f_out.writelines(f_in)
-            f_out.close()
-            f_in.close()
+            try:
+                subprocess.check_call(["gzip", tmp_output_path])
+            except subprocess.CalledProcessError as e:
+                logger.error("Unable to zipp: {}".format(tmp_output_path))
```

Comment:
Do you use log.error(e) to get rid of the traceback? Or do we want to have it? If yes, just use `log.exception(message)`